test/services/cache/ExportsDeclarationBuilder.scala

```diff
@@ -102,6 +102,10 @@ trait ExportsDeclarationBuilder {
     m.copy(locations = m.locations.copy(goodsLocation = Some(goodsLocation)))
   }
 
+  def withoutGoodsLocation(): ExportsDeclarationModifier = { m =>
+    m.copy(locations = m.locations.copy(goodsLocation = None))
+  }
```

Comment:
More readable than `_` though.

Reply:
You can't use `_` because you're invoking `m` two times.

Change to `model` it's not a big deal in general, but if you look at the code you don't need to think what's this. It's non blocking comment of course.

Reply:
I will change it.    I was just making the point that scalas use of _ is also dumb (even if it can't be used here_